functions/exercise21/scriptSpec.js

```diff
@@ -0,0 +1,13 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals getPossibleSubset*/
+
+describe("Getting all possible subset with a fixed length combinations in an array", (() => {
+    it("If the array [1, 2, 3] we will get all subsets of length 2, we get subsets [2, 1], [3, 1], [3, 2]", (() => {
+        var subsets = getPossibleSubset([1, 2, 3], 2);
+        //var subsets = [[3, 2], [2, 1], [3, 1]];
+        expect(subsets.length).toEqual(3);
+        expect(subsets.filter(x => x.length === 2 && x.includes(2) && x.includes(3)).length).toBeTruthy();
```

Comment:
The 3 following lines are very similar. Please extract a function to encapsulate this behaviour and call it 3 times. Later it would be useful in some other tests.